gems/pending/Gemfile

```diff
@@ -24,7 +24,7 @@ gem "fog",                     "~>2.0.0.pre.0",     :require => false
 gem "fog-core",                "!=1.31.1",          :require => false
 gem "httpclient",              "~>2.5.3",           :require => false
 gem "kubeclient",              "=0.8.0",            :require => false
-gem "hawkular-client",                    :require => false, :git => "git@github.com:hawkular/hawkular-client-ruby.git", :branch => "hawkular-all"
+gem "hawkular-client",                    :require => false, :git => "git@github.com:hawkular/hawkular-client-ruby.git", :branch => "master"
```

Comment:
tiny nit: afaik, for master branch you don't have to specify it. 
btw did you already release the gem? if yes, it can be then pulled instead from rubygems instead of the repo here